back/lib/tasks/single_use/20240613_rename_gv_profile.rake

```diff
@@ -5,7 +5,7 @@ namespace :gv_transition do
     Tenant.safe_switch_each do |tenant|
       default_moderators = User.where(email: %w[moderator@citizenlab.co moderator@govocal.com])
       default_moderators.each do |moderator|
-        if moderator.first_name&.casecmp('citizenlab')&.zero?
+        if moderator.first_name&.downcase == 'citizenlab'
```

Comment:
Also tested without the ampersand: https://app.circleci.com/pipelines/github/CitizenLabDotCo/citizenlab/195683/workflows/f437b919-e67e-4fd4-976b-c30ac1ba7f91